alf/utils/dist_utils.py

```diff
@@ -695,16 +701,19 @@ def _get_transformed_builder(obj: td.TransformedDistribution):
     return new_builder, new_params
 
 
-def _builder_affine_transformed(base_builder, loc_, scale_, **kwargs):
-    # 'loc' and 'scale' may conflict with the names in kwargs. So we add suffix '_'.
-    return AffineTransformedDistribution(base_builder(**kwargs), loc_, scale_)
+def _builder_affine_transformed(base_builder, params_, transforms_params_):
+    return AffineTransformedDistribution(
+        base_builder(**params_), **transforms_params_)
 
 
 def _get_affine_transformed_builder(obj: AffineTransformedDistribution):
     builder, params = _get_builder(obj.base_dist)
-    new_builder = functools.partial(_builder_affine_transformed, builder,
-                                    obj.loc, obj.scale)
-    return new_builder, params
+    new_builder = functools.partial(_builder_affine_transformed, builder)
+    new_params = {
+        "params_": params,
+        "transforms_params_": obj.transforms[0].params
+    }
+    return new_builder, new_params
```

Comment:
What is the reason for making this change?

Reply:
> What is the reason for making this change?

Need an easy way to extract all transformed params, including the special AffineTransformedDistribution. @QuantuMope is writing the code to split an action distribution into two for the one-bit movement PR. When we split a distribution, we might also need to split its transformed params.

Reply:
Shouldn't the original code for this part still work?